core/src/main/java/hudson/ExpressionFactory2.java

```diff
@@ -72,7 +72,7 @@ public Object evaluate(JellyContext context) {
                 CURRENT_CONTEXT.set(context);
                 JexlContext jexlContext = new JellyJexlContext( context );
                 return expression.evaluate(jexlContext);
-            } catch (AcegiSecurityException e) {
+            } catch (AccessDeniedException2 e) {
```

Comment:
There is no replacement for `AcegiSecurityException`. AFAIK the purpose of this class is to nicely handle 403s, so this should be good enough.